TwMapView.js

```diff
@@ -25,7 +25,7 @@ export class TwMapView extends React.Component {
 				<MapView
 					style={styles.map}
 					initialRegion={this.state.region}
-					onRegionChange={this.onRegionChange}
+					onRegionChange={() => this.onRegionChange.bind(this)}
```

Comment:
Just out of curiosity, does "this" refer to the MapView or the TwMapView component?